### PR TITLE
Route support for OpenShift deployments

### PR DESCRIPTION
**Pull request by gasgithub**
Added support for Route object to be used in case of OpenShift deployments instead of Ingress.

Changes:
- route.yaml in templates
- new section in values.yaml
- updated Notes to get the route URL

Based on previous comments:
Restored default settings in values.yaml. Set default to not create route. Bumped version to pass tests.

### CODE REVIEW

File: charts/sonarqube/values.yaml [hunk 1]
```diff
@@ -86,6 +86,15 @@ ingress:
   #   hosts:
   #     - chart-example.local
 
+# Create Route for OpenShift deployments. Set enabled=true if you want Route to be automatically generated based on service and namespace names.
+route:
```
Comment (tobias-trabelsi-sonarsource) [inline charts/sonarqube/values.yaml:90]: I think the new values should at least be mentioned in the `README.md`. don't you think?

**Review by tobias-trabelsi-sonarsource — CHANGES_REQUESTED** [review]

File: charts/sonarqube/values.yaml (hunk 1 above)
Comment (gasgithub) [reply charts/sonarqube/values.yaml:90]: I added config params to README, although I think that most users will read the descriptions from the `values.yaml` file anyway.

**Comment by tobias-trabelsi-sonarsource** [comment]
Hi @gasgithub ,

can you rebase and bump the chart version once more? after that i can merge your PR

File: charts/sonarqube/values.yaml (hunk 1 above)
Comment (os-learner) [reply charts/sonarqube/values.yaml:90]: I agree with you both. README.md is the main file anyone will read.

**Comment by os-learner** [comment]
Thanks all.

**Pull request closed without merging**